app/build.gradle

```diff
@@ -68,6 +68,8 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
     implementation project(':sdk')
+    implementation project(':transfer')
```

Comment:
Not sure we want to add it right now to the sample app, @YohayBar WDYT ?
Should be tested with QA also, please provide a version to Avishy so he can test.

Reply:
I think this should move to kin-ecosystem-sdk if we want to enable it by default to all partners

Reply:
i added it to sample app for testing and showing how to implement  it in your side.
this is up to you if needs to be included

Reply:
As we talked it can be in the SDK module